backend/organization/utility/email.py

```diff
@@ -211,6 +211,81 @@ def send_project_follower_email(user, project_follower, notification):
     )
 
 
+def send_organization_follower_email(user, organization_follower, notification):
+    lang_code = get_user_lang_code(user)
+
+    following_user_full_name = (
+        organization_follower.user.first_name
+        + " "
+        + organization_follower.user.last_name
+    )
+
+    subjects_by_language = {
+        "en": "{} now follows {} on Climate Connect".format(
+            following_user_full_name, organization_follower.organization.name
+        ),
+        "de": "{} folgt jetzt {} auf Climate Connect".format(
+            following_user_full_name, organization_follower.organization.name
```

Comment:
I think we should use the name translation here.

Reply:
Uses name translation now.